src/main/webapp/js/common.js

```diff
@@ -99,21 +99,24 @@ var DISPLAY_INPUT_FIELDS_EXTRA = 'There are too many fields.';
 var DISPLAY_INPUT_FIELDS_MISSING = 'There are missing fields.';
 var DISPLAY_GOOGLEID_INVALID = 'GoogleID should only consist of alphanumerics, fullstops, dashes or underscores.';
 var DISPLAY_EMAIL_INVALID = 'The e-mail address is invalid.';
-var DISPLAY_NAME_INVALID = 'Name should only consist of alphanumerics or hyphens, apostrophes, fullstops, commas, slashes, round brackets<br> and not more than 40 characters.';
+var DISPLAY_NAME_INVALID = 'Name should only consist of alphanumerics or hyphens, apostrophes, fullstops, '
+                         + 'commas, slashes, round brackets<br> and not more than 40 characters.';
```

Comment:
Aligning `=` and `+` does not reflect the hierarchy. I prefer if the `+` appears right of the `=`.